server.js

```diff
@@ -21,7 +14,23 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Netflix data!");
+});
+
+app.get('/movies', (req, res) => {
+  res.status(200).json(netflixData);
+});
+
+app.get("/movies/id/:id", (req,res) => {
+  const { id } = req.params
+  const showId = netflixData.find(show => show.show_id === +id)
```

Comment:
You've made it according blue requirements with one endpoint giving an array and one endpoint giving a unique item. Great work!